streaming/base/format/base/writer.py

```diff
@@ -38,27 +60,43 @@ class Writer(ABC):
     format: str = ''  # Name of the format (like "mds", "csv", "json", etc).
 
     def __init__(self,
-                 dirname: str,
+                 *,
+                 local: Optional[str] = None,
+                 remote: Optional[str] = None,
+                 keep: bool = False,
                  compression: Optional[str] = None,
                  hashes: Optional[List[str]] = None,
                  size_limit: Optional[int] = 1 << 26,
                  extra_bytes_per_shard: int = 0,
                  extra_bytes_per_sample: int = 0) -> None:
+        if local:
+            pass
+        elif remote:
```

Comment:
can we please write specifically `if remote is None` ?

Reply:
empty string should be falsy